client/views/windows/settings.tpl

```diff
@@ -4,6 +4,15 @@
 <div class="container">
 	<h1 class="title">Settings</h1>
 
+	<div class="row">
+		<div class="col-sm-6">
+			<label class="opt">
+				<input type="checkbox" name="advanced">
+				Show advanced settings
```

Comment:
"Advanced settings" only maybe?
Also maybe display this at the end? If it's advanced and not useful to everyone, I'm thinking it doesn't need to be the first thing everyone sees. But not married to that idea either.

Reply:
> "Advanced settings" only maybe?

Sure, will fix

> Also maybe display this at the end? If it's advanced and not useful to everyone, I'm thinking it doesn't need to be the first thing everyone sees. But not married to that idea either.

Hmmm, can do. But should it be behind the sessions and the password reset? I was thinking this position is more of a "over all the settings" which can stay when the settings are turned into tabs.